docs/working-remotely-ssh.md

```diff
@@ -0,0 +1,49 @@
+# Working remotely with Visual Studio Code
+
+It is possible to work remotely on a machine containing the required opensafely software using Visual Studio Code.
```

Comment:
nit: "opensafely software" here should probably be "OpenSAFELY software".

(Aside: Across the documentation, I'm not sure that we have a strong convention for distinguishing "all the software you need to run OpenSAFELY" from "the OpenSAFELY command-line tool: `opensafely`" — which is also software and a subset of the first.)